docs/basic_training/channels.md

```diff
@@ -693,64 +693,162 @@ Files containing JSON content can also be parsed:
 
 This can also be used as a way to parse YAML files:
 
-```groovy linenums="1"
-import org.yaml.snakeyaml.Yaml
+=== "Source code"
 
-def f = file('data/meta/regions.yml')
-def records = new Yaml().load(f)
+    ```groovy linenums="1"
+    import org.yaml.snakeyaml.Yaml
 
+    def f = file('data/meta/regions.yml')
+    def records = new Yaml().load(f)
 
-for (def entry : records) {
-    log.info "$entry.patient_id -- $entry.feature"
-}
-```
+
+    for (def entry : records) {
+        log.info "$entry.patient_id -- $entry.feature"
+    }
+    ```
+
+=== "data/meta/regions.yml"
+
+    ```yml
+    - patient_id: ATX-TBL-001-GB-01-105
+    region_id: R1
+    feature: pass_vafqc_flag
+    pass_flag: "TRUE"
+    - patient_id: ATX-TBL-001-GB-01-105
+    region_id: R1
+    feature: pass_stripy_flag
+    pass_flag: "TRUE"
+    - patient_id: ATX-TBL-001-GB-01-105
+    region_id: R1
+    feature: pass_manual_flag
+    pass_flag: "TRUE"
+    - patient_id: ATX-TBL-001-GB-01-105
+    region_id: R1
+    feature: other_region_selection_flag
+    pass_flag: "TRUE"
+    - patient_id: ATX-TBL-001-GB-01-105
+    region_id: R1
+    feature: ace_information_gained
+    pass_flag: "TRUE"
+    - patient_id: ATX-TBL-001-GB-01-105
+    region_id: R1
+    feature: concordance_flag
+    pass_flag: "TRUE"
+    - patient_id: ATX-TBL-001-GB-01-105
+    region_id: R2
+    feature: pass_vafqc_flag
+    pass_flag: "TRUE"
+    - patient_id: ATX-TBL-001-GB-01-105
+    region_id: R2
+    feature: pass_stripy_flag
+    pass_flag: "TRUE"
+    - patient_id: ATX-TBL-001-GB-01-105
+    region_id: R2
+    feature: pass_manual_flag
+    pass_flag: "TRUE"
+    - patient_id: ATX-TBL-001-GB-01-105
+    region_id: R2
+    feature: other_region_selection_flag
+    pass_flag: "TRUE"
+    - patient_id: ATX-TBL-001-GB-01-105
+    region_id: R2
+    feature: ace_information_gained
+    pass_flag: "TRUE"
+    - patient_id: ATX-TBL-001-GB-01-105
+    region_id: R2
+    feature: concordance_flag
+    pass_flag: "TRUE"
+    - patient_id: ATX-TBL-001-GB-01-105
+    region_id: R3
+    feature: pass_vafqc_flag
+    pass_flag: "TRUE"
+    - patient_id: ATX-TBL-001-GB-01-105
+    region_id: R3
+    feature: pass_stripy_flag
+    pass_flag: "FALSE"
```

Comment:
I think that we can just include the file here..
```suggestion
    --8<-- "../../nf-training/data/meta/regions.yml"
```

Reply:
Great idea! 🤩